src/controllers/hid/hidcontroller.cpp

```diff
@@ -277,6 +277,38 @@ void HidController::send(QByteArray data, unsigned int reportID) {
     data.prepend(reportID);
 
     int result = hid_write(m_pHidDevice, (unsigned char*)data.constData(), data.size());
+    if (result == -1) {
```

Comment:
should this be a constant (constexpr in anonymous namespace)?

Reply:
I didn't changed the code of HidController::send

Reply:
I think the diff is visualizing this incorrect. I added a new method after HidController::send with mostly the same content. The diff marks my code as unchanged and the original code as changed.